app/internal/page/share_code.go

```diff
@@ -69,6 +66,13 @@ func (s *ShareCodeSender) SendCertificateProvider(ctx context.Context, template
 }
 
 func (s *ShareCodeSender) SendAttorneys(ctx context.Context, appData AppData, lpa *Lpa) error {
+	if err := s.sendTrustCorporation(ctx, notify.TrustCorporationInviteEmail, appData, lpa, lpa.Attorneys.TrustCorporation, false); err != nil {
+		return err
+	}
+	if err := s.sendTrustCorporation(ctx, notify.ReplacementTrustCorporationInviteEmail, appData, lpa, lpa.ReplacementAttorneys.TrustCorporation, true); err != nil {
```

Comment:
I'm using the same templates at the moment, but I'm sure they will be different when these emails are looked at